homeassistant/components/husqvarna_automower/sensor.py

```diff
@@ -422,12 +416,9 @@ class WorkAreaSensorEntityDescription(SensorEntityDescription):
     WorkAreaSensorEntityDescription(
         key="last_time_completed",
         translation_key_fn=_work_area_translation_key,
-        exists_fn=lambda data: data.last_time_completed_naive is not None,
+        exists_fn=lambda data: data.last_time_completed is not None,
         device_class=SensorDeviceClass.TIMESTAMP,
-        value_fn=lambda data: naive_to_aware(
-            data.last_time_completed_naive,
-            ZoneInfo(str(dt_util.DEFAULT_TIME_ZONE)),
-        ),
+        value_fn=lambda data: data.last_time_completed,
```

Comment:
Probably doesn’t matter but could use attrgetter here

Reply:
I also changed the other occurrences in this platform